boards/xtensa/esp32/doc/esp32.rst

```diff
@@ -0,0 +1,159 @@
+.. _esp32:
+
+ESP32
+#####
+
+Overview
+********
+
+From Wikipedia::
```

Comment:
I'd change this to be a footnoted reference rather an a block quote with the angle brackets:

     ESP32 is a series of low cost, low power system on a chip microcontrollers
     with integrated Wi-Fi & dual-mode Bluetooth.  The ESP32 series employs a
     Tensilica Xtensa LX6 microprocessor in both dual-core and single-core
     variations.  ESP32 is created and developed by Espressif Systems, a
     Shanghai-based Chinese company, and is manufactured by TSMC using their 40nm
     process. [1]_